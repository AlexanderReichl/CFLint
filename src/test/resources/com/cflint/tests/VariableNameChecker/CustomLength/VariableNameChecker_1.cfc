<!--- scoped use of name is detected, no unused arg flagged--->
<cfcomponent>
<cffunction name="helloWorld" access="public" returntype="string">
  <cfscript>
    var x = 1;
    var yyy = 1;
  </cfscript>
</cffunction>
</cfcomponent>