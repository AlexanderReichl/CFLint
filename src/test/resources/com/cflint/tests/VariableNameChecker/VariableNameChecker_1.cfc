<!--- scoped use of name is detected, no unused arg flagged--->
<cfcomponent>
<cffunction name="helloWorld" access="public" returntype="string">
  <cfscript>
    var x = 1;
    ARGUMENTS.ALLcAPSWAYTOOLONGETETCETC;
    var temp = 1;
    var WayMuchTooWordyExample = '';
    var objBadName = '';
  </cfscript>
</cffunction>
</cfcomponent>