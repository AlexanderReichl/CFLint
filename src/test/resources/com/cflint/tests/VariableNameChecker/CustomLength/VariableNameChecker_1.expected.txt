[ {
  "severity" : "ERROR",
  "id" : "VAR_TOO_SHORT",
  "message" : "VAR_TOO_SHORT",
  "category" : "CFLINT",
  "abbrev" : "VT",
  "locations" : [ {
    "file" : "src/test/resources/com/cflint/tests/VariableNameChecker/CustomLength/VariableNameChecker_1.cfc",
    "fileName" : "VariableNameChecker_1.cfc",
    "function" : "helloWorld",
    "column" : "7",
    "line" : "5",
    "message" : "Variable x should be longer than 4 characters.",
    "variable" : "x",
    "expression" : "var x = 1"
  } ]
}, {
  "severity" : "ERROR",
  "id" : "VAR_TOO_SHORT",
  "message" : "VAR_TOO_SHORT",
  "category" : "CFLINT",
  "abbrev" : "VT",
  "locations" : [ {
    "file" : "src/test/resources/com/cflint/tests/VariableNameChecker/CustomLength/VariableNameChecker_1.cfc",
    "fileName" : "VariableNameChecker_1.cfc",
    "function" : "helloWorld",
    "column" : "7",
    "line" : "6",
    "message" : "Variable yyy should be longer than 4 characters.",
    "variable" : "yyy",
    "expression" : "var yyy = 1"
  } ]
} ]